src/components/EntityPage/File/index.js

```diff
@@ -230,43 +322,54 @@ const FileEntity = ({ api, fileId }) => {
       sqon={buildSqonForIds([fileId])}
       transform={data => _.get(data, 'data.file.hits.edges[0].node')}
     >
-      {file => (
-        <Container>
-          <EntityTitleBar>
-            <EntityTitle
-              icon="file"
-              title={fileId}
-              tags={file.isLoading || true ? [] : getTags(file.data)}
-            />
-          </EntityTitleBar>
-          <EntityActionBar>Share Button</EntityActionBar>
-          <EntityContent>
-            <EntityContentSection title="File Properties">
-              <Row style={{ width: '100%' }}>
-                <Column style={{ flex: 1, paddingRight: 15, border: 1 }}>Summary Table 1</Column>
-                <Column style={{ flex: 1, paddingLeft: 15, border: 1 }}>Summary Table 2</Column>
-              </Row>
-            </EntityContentSection>
-            <EntityContentDivider />
-            <EntityContentSection title="Associated Participants/Biospecimens">
-              Participant and Biospeciment Table Here
-              <BaseDataTable
-                columns={mockColumns.map(col => ({ ...col, ...{ show: true } }))}
-                loading={false}
-                data={mockData}
+      {file => {
+        return file.isLoading ? (
+          <div>Loading</div>
+        ) : (
+          <Container>
+            <EntityTitleBar>
+              <EntityTitle
+                icon="file"
+                title={fileId}
+                tags={file.isLoading || true ? [] : getTags(file.data)}
               />
-            </EntityContentSection>
-            <EntityContentDivider />
-            <EntityContentSection title="Associated Experimental Strategies">
-              Experimental Strategy Table Here
-            </EntityContentSection>
-            <EntityContentDivider />
-            <EntityContentSection title="Sequencing Read Properties">
-              Read Property Numbers Here
-            </EntityContentSection>
-          </EntityContent>
-        </Container>
-      )}
+            </EntityTitleBar>
+            <EntityActionBar>Share Button</EntityActionBar>
+            <EntityContent>
+              <EntityContentSection title="File Properties">
+                <Row style={{ width: '100%' }}>
+                  <Column style={{ flex: 1, paddingRight: 15, border: 1 }}>
+                    <SummaryTable rows={summaryTable1Data(file.data)} />
```

Comment:
@ciaranschutte Not a big issue but when I read `summaryTable1Data` and `summaryTable2Data` I don't really know what kind of data is in there.

Reply:
@nyanofthemoon Do you think it would be better to create the data once and have a better name like so: 

```

const fileEntitySummary = data => {
  const participants = data.participants.hits.edges[0].node;
  const study = participants.study;
  const biospecimens = participants.biospecimens.hits.edges[0].node;
  return [
    {
      title: 'External ID:',
      summary: <ExternalLink href={``}>data.external_id</ExternalLink>,
    },
    { title: 'Name:', summary: data.file_name },
    { title: 'Study:', summary: `${study.short_name}(${study.kf_id})` },
    { title: 'Access:', summary: data.controlled_access ? 'Controlled' : '' },
    { title: 'Consent Codes:', summary: biospecimens.dbgap_consent_code },
    {
      title: 'Harmonized Data:',
      summary: data.is_harmonized ? 'Yes' : 'No',
    },
    { title: 'Reference Genome:', summary: data.reference_genome },
    {
      title: 'Experimental Strategy:',
      summary: data.experiment_strategies.map(strategies => <div>{strategies}</div>),
    },
    { title: 'Data Type:', summary: data.data_type },
    { title: 'File Format:', summary: data.file_format },
    { title: 'Size:', summary: formatToGB(data.size) },
  ];
};
```

then in render, pull out the data for each data: 
```
 {file => {
        if (file.isLoading) {
          return <div>Loading</div>;
        } else {
          const fileEntitySummaryData = fileEntitySummary(file.data);
          console.log('data', fileEntitySummaryData);
          const summaryTable1 = fileEntitySummaryData.filter(d =>
            [
              'Harmonized Data:',
              'Reference Genome:',
              'Experimental Strategy:',
              'Experimental Strategy:',
              'Data Type:',
              'File Format:',
              'Size:',
            ].includes(d.title),
          );
          console.log('summary table1', summaryTable1);
          //const summaryTable2 = fileEntitySummaryData.filter(fileEntitySummaryData,)
          return (
            <Container>
              <EntityTitleBar>
                <EntityTitle
.......
```

Reply:
Appears more clear and easier to test methinks.